usvm-python/usvm-python-main/src/main/kotlin/org/usvm/machine/interpreters/concrete/ConcretePythonInterpreter.kt

```diff
@@ -162,7 +162,18 @@ object ConcretePythonInterpreter {
     }
 
     fun allocateVirtualObject(virtualObject: VirtualPythonObject): PyObject {
-        val ref = pythonAdapter.allocateVirtualObject(virtualObject)
+        /*
+         * Usage example:
```

Comment:
I believe, now this `usage example` should be modified to use your new auxiluary methods.

Reply:
Done